extensions/widgets/segmented/segmented.lcb

```diff
@@ -108,17 +102,17 @@ pSegmentDisplay(enum): The style of the segment display.
 -"label": Show the chosen labels
 
 Description:
-Sets the segment display style of the control to either icon or label.
+Use the <itemStyle> property to display either icons or labels in the segments.
 **/
-property segmentDisplay		get mSegmentDisplay		set setSegmentDisplay
-metadata segmentDisplay.editor		is "com.livecode.pi.enum"
-metadata segmentDisplay.options		is "text,icon"
-metadata segmentDisplay.default		is "text"
-metadata segmentDisplay.label		is "Display Style"
+property itemStyle		get mSegmentDisplay		set setSegmentDisplay
+metadata itemStyle.editor		is "com.livecode.pi.enum"
+metadata itemStyle.options		is "text,icon"
```

Comment:
At the moment, I'm using "text,icons,both" for the navbar.  Should I change that to "text,icon,both", or would you prefer to change this to "text,icons"?